frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -76,11 +81,16 @@ private void startCountdown() {
 			labelCountdown.setValue(Integer.toString(second));
 			second--;
 
-			if (second < 0) {
+			if (this.getTeamLives() == 0) {
+				timeline.stop();
+				showLevelSuccessDialog.setValue(false);
+			}
+			if ((this.getTeamLives() != 0) && (second < 0)) {
 				timeline.stop();
 				labelCountdown.setValue("Time over");
```

Comment:
Kannst du das `if (this.getTeamLives() == 0)` nur um die labelCountdown Zuweisung setzen 🤔 ? ... also ich frage mich, ob man das vereinfachen kann:
```
if (this.getTeamLives() == 0) || second < 0) {
 timeline.stop();
 showLevelSuccessDialog.setValue(false);
}

if ((this.getTeamLives() != 0) && (second < 0)) {
 labelCountdown.setValue("Time over");
}
```

Reply:
Habe es etwas vereinfachen können:
if (!this.teamHasLivesLeft()) {
				timeline.stop();
			}
			if (this.teamHasLivesLeft() && (second < 0)) {
				timeline.stop();
				labelCountdown.setValue("Time over");
				showLevelSuccessDialog.setValue(true);
			}